### PR TITLE
Allow building with Stack

### PR DESCRIPTION
**Pull request by googleson78**

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name:                prob-school
```
Comment (googleson78) [inline package.yaml:1]: Could you fill in (or delete) these yourself, I'm not sure what it should say here?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-$ cabal install Chart-cairo
-$ cabal install data-default
-$ cabal install logfloat
-$ cabal install erf
-$ cabal install transformers
-$ cabal install cassava
-$ cabal install pretty-show
-$ ghci Casino.hs
-*Casino> main
+stack ghci
```
Comment (ccshan) [inline README.md:5]: Should this say `stack ghci Casino.hs` ?

Comment (googleson78) [reply README.md:5]: You could do it, but all the modules are available to you once you do `stack ghci`, so you could instead do just `import Casino`

Comment (googleson78) [reply README.md:5]: I just forgot to write this in!

**Comment by ccshan** [comment]
Thanks!!

**Pull request merged**